addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -760,47 +760,44 @@
 
 <template id='lesson_card' name="Lesson Card">
     <div class="card w-100 o_wslides_lesson_card mb-4">
-            <t t-if="slide.is_new_slide and not channel_progress[slide.id].get('completed')" t-call="website_slides.course_card_information"/>
+        <t t-if="slide.is_new_slide and not channel_progress[slide.id].get('completed')" t-call="website_slides.course_card_information"/>
+        <t t-set="default_slide_background" t-value="website.image_url(channel, 'image_1024') if channel.image_1024 else '/website_slides/static/src/img/channel-%s-default.jpg' % ('training' if channel.channel_type == 'training' else 'documentation')"/>
         <t t-set="can_access" t-value="slide.is_preview or channel.is_member or channel.can_publish"/>
-
-        <t t-if="slide.image_1024">
-            <t t-set="lesson_image" t-value="website.image_url(slide, 'image_1024')"/>
-            <a t-if="can_access" t-attf-href="/slides/slide/#{slug(slide)}#{query_string}" t-title="slide.name">
-                <div class="card-img-top border-bottom" t-attf-style="padding-top: 50%; background-image: url(#{lesson_image}); background-size: cover; background-position:center"/>
-            </a>
-            <div t-else="" class="card-img-top border-bottom" t-attf-style="padding-top: 50%; background-image: url(#{lesson_image}); background-size: cover; background-position:center"/>
-        </t>
-        <t t-else="">
-            <a t-if="can_access" t-attf-href="/slides/slide/#{slug(slide)}#{query_string}" t-title="slide.name">
-                <div class="o_wslides_card_default_background card-img-top border-bottom" t-attf-style="padding-top: 50%;"/>
-            </a>
-            <div t-else="" class="o_wslides_card_default_background card-img-top border-bottom" t-attf-style="padding-top: 50%;"/>
-        </t>
+        <a t-if="can_access" t-attf-href="/slides/slide/#{slug(slide)}#{query_string}" t-title="slide.name" style="height:120px">
+            <div t-if="slide.image_1920" t-field="slide.image_1920" t-options="{'widget': 'image', 'style': 'height: 100%; width: 100%; object-fit: cover', 'preview_image': 'image_512'}" class="h-100"/>
+            <div t-else="">
+                <img t-att-src="default_slide_background" class="img img-fluid" style="height: 1100%; width: 100%; object-fit: cover"/>
+            </div>
+        </a>
+        <div t-else="" style="height:120px">
+            <img t-att-src="default_slide_background" class="img img-fluid" style="height: 100%; width: 100%; object-fit: cover"/>
+        </div>
```

Comment:
Could get rid of some style attributes here and replace it by proper scss if you are up for it :)